packages/core/lib/src/core_widget_factory.dart

```diff
@@ -1150,14 +1158,103 @@ class WidgetFactory {
     return baseUrl.resolveUri(uri).toString();
   }
 
+  /// Builds custom widget for div elements with display: flex from [meta]
+  BuildOp _flexOp(BuildMetadata meta) {
+    return BuildOp(
```

Comment:
This op should be in its own file I think, probably `packages/core/lib/src/internal/ops/flex.dart`. Because of the complexity, flex implementation will grow to support all kind of use cases, the factory file is already too long.